build.gradle

```diff
@@ -6,17 +6,31 @@ group 'org.example'
 version '1.0-SNAPSHOT'
 
 repositories {
-    mavenCentral()
+    //mavenCentral()
+    maven {
+        //url 'https://mytest.artifactory.com/artifactory/urs-mvn'
+        //url = uri("https://${artifactory_for_maven}/${artifactory_repo_for_maven}")
+        url = uri("${artifactory_for_maven}/${artifactory_repo_for_maven}")
+        credentials {
+            username  "${artifactory_user}"
+            password  "${artifactory_password}"
+        }
+        metadataSources {
+            mavenPom()
+            artifact()
+            gradleMetadata()
+        }
+    }
 }
 
 dependencies {
-    implementation 'org.codehaus.groovy:groovy-all:3.0.0'
+    implementation 'org.apache.groovy:groovy:4.0.1'
+    testImplementation 'junit:junit:4.3'
```

Comment:
I explicitly used only TestNG in the project to avoid JUnit. Have you added anything specific for JUnit?

Reply:
No I did not add any JUnit tests